.github/workflows/merge.yaml

```diff
@@ -20,4 +20,4 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       - name: Push Windows logger agent
-        run: make push-logger-windows
\ No newline at end of file
+        run: DOCKER_REGISTRY=brigadecore make push-logger-windows
```

Comment:
I think this should be `DOCKER_ORG= brigadecore`

Reply:
Ah, you're right.  While both would work here, DOCKER_ORG is definitely more proper.  Fixed!